interface/patient_file/report/custom_report.php

```diff
@@ -817,7 +821,10 @@ function postToGet($arin)
 if ($PDF_OUTPUT) {
     $content = getContent();
     $ptd = getPatientData($pid, "fname,lname");
-    $fn = strtolower($ptd['fname'] . '_' . $ptd['lname'] . '_' . $pid . '_' . xl('report') . '.pdf');
+    // escape names for pesky periods hyphen etc.
+    $esc = $ptd['fname'] . '_' . $ptd['lname'];
```

Comment:
Won't this break for internationalization (ie. chinese names etc.)? Why not just use the pid? Or is the file name needed for display (figured may be case considering translating report)?
fyi, note there are also some available functions for escaping stuff in filename:
https://github.com/openemr/openemr/blob/master/library/sanitize.inc.php#L46-L67